pyscf/pbc/scf/test/test_khf_ksym.py

```diff
@@ -43,7 +43,7 @@ def make_primitive_cell(mesh, spin=0):
 
 def setUpModule():
     global cell, He, nk, kmf0, kumf0, kmf_ksymm, kumf_ksymm
-    cell = make_primitive_cell([16]*3)
+    cell = make_primitive_cell([17]*3)
```

Comment:
16 will allow to check some non-symmorphic symmetry operations, and 17 will remove those operations. I should add more checks.